plugins/org.polarsys.capella.transition.system2subsystem.tests.ju/META-INF/MANIFEST.MF

```diff
@@ -28,3 +28,4 @@ Export-Package: org.polarsys.capella.transition.system2subsystem.tests,
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
+Import-Package: org.polarsys.capella.core.transition.system.topdown.constants
```

Comment:
to remove.

subsystem must not depend on topdown.
and Import-Package shall not be used, only Require-Bundle.